### PR TITLE
Drop deprecated TypeScript properties: `sonar.typescript.lcov.reportPaths`, `sonar.typescript.node`, `sonar.typescript.eslint.reportPaths`

### LINKED ISSUE #2758: Drop deprecated TypeScript properties
- sonar.typescript.node
- sonar.typescript.lcov.reportPaths

### PR DESCRIPTION
**Pull request by yassin-kammoun-sonarsource**
Fixes #2758

### CODE REVIEW

File: nodejs-utils/src/main/java/org/sonarsource/nodejs/NodeCommandBuilderImpl.java [hunk 1]
```diff
@@ -186,17 +185,10 @@ private String getVersion(String nodeExecutable) throws NodeCommandException {
   }
 
   private String retrieveNodeExecutableFromConfig(@Nullable Configuration configuration) throws NodeCommandException, IOException {
-    if (configuration != null && (configuration.hasKey(NODE_EXECUTABLE_PROPERTY) || configuration.hasKey(NODE_EXECUTABLE_PROPERTY_TS))) {
+    if (configuration != null && (configuration.hasKey(NODE_EXECUTABLE_PROPERTY))) {
```
Comment (vilchik-elena) [inline nodejs-utils/src/main/java/org/sonarsource/nodejs/NodeCommandBuilderImpl.java:188]: remove redundant parentheses

File: nodejs-utils/src/main/java/org/sonarsource/nodejs/NodeCommandBuilderImpl.java [hunk 2]
```diff
@@ -186,17 +185,10 @@ private String getVersion(String nodeExecutable) throws NodeCommandException {
...
       String usedProperty = "";
 
-      if (configuration.hasKey(NODE_EXECUTABLE_PROPERTY_TS)) {
-        LOG.warn("The use of " + NODE_EXECUTABLE_PROPERTY_TS + " is deprecated, use "
-          + NODE_EXECUTABLE_PROPERTY + " instead.");
-        usedProperty = NODE_EXECUTABLE_PROPERTY_TS;
-        nodeExecutable = configuration.get(NODE_EXECUTABLE_PROPERTY_TS).get();
-      }
-
       if (configuration.hasKey(NODE_EXECUTABLE_PROPERTY)) {
```
Comment (vilchik-elena) [inline nodejs-utils/src/main/java/org/sonarsource/nodejs/NodeCommandBuilderImpl.java:192]: it's always true

File: nodejs-utils/src/main/java/org/sonarsource/nodejs/NodeCommandBuilderImpl.java [hunk 3]
```diff
@@ -186,17 +185,10 @@ private String getVersion(String nodeExecutable) throws NodeCommandException {
   }
 
   private String retrieveNodeExecutableFromConfig(@Nullable Configuration configuration) throws NodeCommandException, IOException {
-    if (configuration != null && (configuration.hasKey(NODE_EXECUTABLE_PROPERTY) || configuration.hasKey(NODE_EXECUTABLE_PROPERTY_TS))) {
+    if (configuration != null && (configuration.hasKey(NODE_EXECUTABLE_PROPERTY))) {
       String nodeExecutable = "";
       String usedProperty = "";
```
Comment (vilchik-elena) [inline nodejs-utils/src/main/java/org/sonarsource/nodejs/NodeCommandBuilderImpl.java:190]: there is no sense in this variable as it's the only property we have

**Review by vilchik-elena — APPROVED** [review]

**Comment by omio-ma** [comment]
where do you guys post for breaking changes like this?

**Pull request merged**